pkg/ddevapp/config.go

```diff
@@ -154,16 +162,20 @@ func (c *Config) Config() error {
 		dir, err := os.Getwd()
 		if err == nil {
 			c.Name = filepath.Base(dir)
+			c.Name = re.ReplaceAllString(c.Name, "")
 		}
 	}
 
 	namePrompt = fmt.Sprintf("%s (%s)", namePrompt, c.Name)
 	// Define an application name.
 	fmt.Print(namePrompt + ": ")
 	c.Name = util.GetInput(c.Name)
+	c.Name = re.ReplaceAllString(c.Name, "")
```

Comment:
We also do this on line 137 - https://github.com/drud/ddev/pull/339/files#diff-1bfb3bdcc7f3b22b48ea905d60583d2dR137

Any reason we are repeating it?